find-closest-facility-from-point/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">Find closest facility from point</string>
+    <string name="san_diego_network_service_url">https://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/ClosestFacility</string>
+    <string name="hospital_symbol_url">https://static.arcgis.com/images/Symbols/SafetyHealth/Hospital.png</string>
+</resources>
```

Comment:
It is good to have a newline at the end of the file, some programs that read files do not read lines without newlines at the end of them.